src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -227,6 +230,14 @@ interface IQueriedMergedTrackCaseData {
     list?: IQueriedCaseData<object>[];
 }
 
+export type GeneticEntity = {
```

Comment:
can you add comments next to the type def lines saying what each value is in the case of gene vs geneset?

Reply:
I don't exactly understand the question: do you want me to comment, e.g., for `geneticEntityId: string|number` number for genes and string for gene sets?

Reply:
yeah exactly, like "entrezGeneId in gene, geneSetId for geneset"